oauthlib/oauth2/rfc6749/request_validator.py

```diff
@@ -166,6 +166,26 @@ def is_within_original_scope(self, request_scopes, refresh_token, request, *args
         """
         return False
 
+    def introspect_token(self, token, token_type_hint, request, *args, **kwargs):
+        """Introspect an access or refresh token.
+
+        Called once introspect token request is validated. This method
+        should return a dictionary with any desired claims associated
+        with the *token*. The implementation can use *token_type_hint*
```

Comment:
You probably want to ensure people will return `None` for unknown tokens instead of raising some error, so that the endpoint return `{"active": false}` instead of an error code for unknown tokens as per spec. Maybe

> with the *token*, or `None` in case the token is unknown.

Reply:
Agree with you